src/Tribe/Views/V2/View.php

```diff
@@ -1765,28 +1765,23 @@ protected function on_page_reset() {
 	protected function get_breadcrumbs() {
 		$context     = $this->context;
 		$breadcrumbs = [];
-		$context_tax = $context->get( TEC::TAXONOMY, false );
+		$taxonomy    = TEC::TAXONOMY;
+		$context_tax = $context->get( $taxonomy, false );
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :no_entry_sign: TRUE, FALSE and NULL must be uppercase; expected "FALSE" but found "false"

<sub>hash: ba7941c89ccd33faa69a5c330fb672ad</sub>